templates/firefox-ios.xliff

```diff
@@ -254,11 +254,15 @@
         <source>Saved Logins</source>
         <note>Settings item for clearing passwords and login data</note>
       </trans-unit>
+      <trans-unit id="Spotlight Index" xml:space="preserve">
+        <source>Spotlight Index</source>
+        <note>Settings item for deleting the iOS search index of browsed webpages</note>
```

Comment:
Please give some short explanation on what "Spotlight" on iOS is, and if that should be translated or not